eng/versioning/version_client.txt

```diff
@@ -6,7 +6,7 @@ com.azure:azure-sdk-parent;1.6.0;1.6.0
 com.azure:azure-client-sdk-parent;1.7.0;1.7.0
 com.azure:azure-ai-anomalydetector;3.0.0-beta.1;3.0.0-beta.2
 com.azure:azure-ai-formrecognizer;3.0.1;3.1.0-beta.1
-com.azure:azure-ai-textanalytics;5.0.0;5.1.0-beta.1
+com.azure:azure-ai-textanalytics;5.0.0;5.1.0-beta.2
```

Comment:
@JimSuplizio  does the dependency version should be `5.1.0-beta.1` instead of `5.0.0`?

Reply:
No, dependency should be the last stable release in this case 5.0.0;